### ISSUE TITLE
LUKS prompt defaults to US keymap despite installing with another keymap

### ISSUE THREAD

**Issue by stalebrim** [getsolus/packages#1544]
### Summary

Despite installing Solus 4.5 with Finnish keymap, the LUKS prompt still defaults to US keymap.

### Steps to reproduce

1. Boot 4.5 ISO
2. Open installer
3. Set up Finnish language and keymap
4. Manually partition so that you've got a non-encrypted boot partition and a LUKS encrypted root partition with a password that uses special characters
5. Install
6. Reboot

### Expected result

The LUKS prompt should accept the password as typed in the installer

### Actual result

The LUKS prompt doesn't accept the password

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Plasma

### System details

Thinkpad T430

### Other comments

Adding vconsole.keymap=fi to boot parameters of the installed system makes the LUKS prompt work as intended.

**Comment by davidjharder** [comment]
calamares discussion: https://github.com/calamares/calamares/issues/1772

**Comment by ermo** [comment]
@silkeh I think maybe we should have the option of adding the user's chosen keymap to the kernel command line with `rd.vconsole.keymap=<whatever>` so it works on first boot after install?

**Comment by silkeh** [comment]
> @silkeh I think maybe we should have the option of adding the user's chosen keymap to the kernel command line with rd.vconsole.keymap=<whatever> so it works on first boot after install?

I don't think it should be an option: it should be set correctly on install when not using QWERTY. I personally have `vconsole.keymap` instead of `rd.vconsole.keymap`, but we can bikeshed that later.

**Comment by ermo** [comment]
@EbonJaeger Any clue how to do this from the installer? Does it need a dedicated post-install script invocation somewhere...?

**Comment by sp0ker** [comment]
I'm still having this problem on the last version, even though I've selected a keymap other than the default US in the installer 😣
